File: project-template/chains/programs/sample.responder.py
import sys, json

# get request and config from the framework
req = json.loads(sys.argv[1])
config = json.loads(sys.argv[2])
params = req['params']

# create response
title = 'sample.responder.py'
name = 'Kimi no na wa?'
if 'name' in params:
    name = params['name']
response = {'title': title, 'name': name}

# show time
print (json.dumps(response))

File: project-template/chains/programs/sample.responder.js
function respond(req, config){
    let params = req['params']
    let title = 'sample.responder.js' 
    let name = 'Kimi no na wa?'
    if('name' in params){
        name = params.name
    }
    return {'title' : title, 'name' : name}
}

module.exports = function(req, config, callback){
    let response = JSON.stringify(respond(req, config)) 
    callback(response)
}

if(require.main == module){
    let req = JSON.parse(process.argv[2])
    let config = JSON.parse(process.argv[3])
    console.log(JSON.stringify(respond(req, config)))
}
